fsproxy/src/test/java/no/unit/identityservice/fsproxy/FsMock.java

```diff
@@ -0,0 +1,462 @@
+package no.unit.identityservice.fsproxy;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.ok;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+import static no.unit.nva.testutils.RandomDataGenerator.randomInteger;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.time.Year;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import no.unit.identityservice.fsproxy.model.course.FsCourse;
+import no.unit.identityservice.fsproxy.model.course.FsCourseItemContainingCourseContainer;
+import no.unit.identityservice.fsproxy.model.course.FsCoursesSearchResult;
+import no.unit.identityservice.fsproxy.model.course.FsSemester;
+import no.unit.identityservice.fsproxy.model.course.FsSubject;
+import no.unit.identityservice.fsproxy.model.person.FsIdNumber;
+import no.unit.identityservice.fsproxy.model.person.FsIdSearchResult;
+import no.unit.identityservice.fsproxy.model.person.FsPerson;
+import no.unit.identityservice.fsproxy.model.person.FsPersonSearchResponse;
+import no.unit.identityservice.fsproxy.model.person.NationalIdentityNumber;
+import no.unit.identityservice.fsproxy.model.staffperson.FsCourseActivity;
+import no.unit.identityservice.fsproxy.model.staffperson.FsRoleToStaffPerson;
+import no.unit.identityservice.fsproxy.model.staffperson.FsRolesToPersonSearchResult;
+import no.unit.identityservice.fsproxy.model.staffperson.FsUriToCourseActivity;
+import no.unit.identityservice.fsproxy.model.staffperson.FsUriToCourseActivityContainer;
+import no.unit.nva.stubs.WiremockHttpClient;
+
+public class FsMock {
```

Comment:
There is a lot of unused code in this class

Reply:
fixed